icu_base.py

```diff
@@ -214,16 +212,10 @@ def build_config_args(self):
             args.append('--disable-tests')
         if not self.options.with_dyload:
             args.append("--disable-dyload")
+        if self.options.with_rpath:
```

Comment:
Why not just use `--enable-rpath` on all platforms and eliminate the need for an option?